diff --git a/verdafile.js b/verdafile.js
index 17f70d5..b5bea11 100644
--- a/verdafile.js
+++ b/verdafile.js
@@ -578,7 +578,7 @@ phony(`full-clean`, async () => {
 async function OtfccBuildOptimize(config, from, to) {
 	const tmpTo = to + ".tmp.otf";
 	const tmpTtx = to + ".tmp.ttx";
-	await run(OTFCCBUILD, from, [`-o`, tmpTo], [`-O3`, `-s`, `--keep-average-char-width`, `-q`]);
+	await run(OTFCCBUILD, from, [`-o`, tmpTo], [`--merge-features`, `--subroutinize`, `--force-cid`, `--ignore-glyph-order`, `-s`, `--keep-average-char-width`, `-q`]);
 	await rm(from);
 	if (config.buildOptions.optimizeWithFilter) {
 		const filterArgs = config.buildOptions.optimizeWithFilter.split(/ +/g);
